code/modules/projectiles/tgprojectile.dm

```diff
@@ -0,0 +1,1296 @@
+//Some debug variables. Toggle them to 1 in order to see the related debug messages. Helpful when testing out formulas.
+#define DEBUG_HIT_CHANCE	FALSE
```

Comment:
use #ifdef for these not if statements

Reply:
CM code, it's just debug statements. will replace to #ifdef